storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/delete/PreparedDeleteObjects.java

```diff
@@ -175,12 +146,24 @@ public Builder(@NonNull StorIOContentResolver storIOContentResolver, @NonNull It
          */
         @NonNull
         public PreparedDeleteObjects<T> prepare() {
+            final ContentResolverTypeDefaults<T> typeDefaults = storIOContentResolver.internal().typeDefaults(type);
+
+            if (mapFunc == null && typeDefaults != null) {
+                mapFunc = typeDefaults.mapToDeleteQuery;
+            }
+
             checkNotNull(mapFunc, "Please specify map function");
 
             if (deleteResolver == null) {
-                deleteResolver = DefaultDeleteResolver.INSTANCE;
+                if (typeDefaults != null && typeDefaults.deleteResolver != null) {
+                    deleteResolver = typeDefaults.deleteResolver;
+                } else {
+                    deleteResolver = DefaultDeleteResolver.INSTANCE;
+                }
             }
 
+            checkNotNull(deleteResolver, "Please specify Delete Resolver");
```

Comment:
My psychiatrist said that I'm paranoid, I want to know has he said it someone else =))